roscala/src/main/scala/coop/rchain/rosette/prim/tuple.scala

```diff
@@ -1,11 +1,22 @@
 package coop.rchain.rosette.prim
 
-import coop.rchain.rosette.{Ctxt, Fixnum, Ob, RblBool, Tuple}
 import coop.rchain.rosette.macros.{checkArgumentMismatch, checkTypeMismatch}
 import coop.rchain.rosette.prim.Prim._
+import coop.rchain.rosette.{Ctxt, Fixnum, Ob, RblBool, Tuple}
 
 object tuple {
 
+  object obNullQ extends Prim {
+    override val name: String = "null?"
+    override val minArgs: Int = 1
+    override val maxArgs: Int = 1
+
+    override def fnSimple(ctxt: Ctxt): Right[PrimError, RblBool] = {
```

Comment:
Why don't use `checkArgumentMismatch` annotation to check if `argvec` actually contains an argument?

Reply:
Done.